app/controllers/common_attribute_states_controller.rb

```diff
@@ -1,71 +1,61 @@
 class CommonAttributeStatesController < ApplicationController
   before_action :auth_user
-  before_action :set_common_attribute_state, only: %i[ show edit update destroy ]
-
-  # GET /common_attribute_states or /common_attribute_states.json
-  def index
-    @common_attribute_states = CommonAttributeState.all
-  end
-
-  # GET /common_attribute_states/1 or /common_attribute_states/1.json
-  def show
-  end
+  before_action :set_room, only: %i[ new create ]
 
   # GET /common_attribute_states/new
   def new
-    @common_attribute_state = CommonAttributeState.new
-  end
+    authorize CommonAttributeState.new
 
-  # GET /common_attribute_states/1/edit
-  def edit
+    @common_attribute_states = CommonAttribute.all.map do |common_attribute|
+      common_attribute.common_attribute_states.new
+    end
   end
 
   # POST /common_attribute_states or /common_attribute_states.json
   def create
-    @common_attribute_state = CommonAttributeState.new(common_attribute_state_params)
+    authorize CommonAttributeState.new
```

Comment:
the same: `authorize CommonAttributeState is enough`, there is no need to add .new